boards/arm/nucleo_f030r8/Kconfig.defconfig

```diff
@@ -10,4 +10,30 @@ if BOARD_NUCLEO_F030R8
 config BOARD
 	default nucleo_f030r8
 
+if UART_CONSOLE
+
+config UART_STM32_PORT_2
+	default y
+
+endif # UART_CONSOLE
+
+# Disable PORT D to allow kernel test cases common and contex
+# to run on nucleo_f030r8
```

Comment:
Is that due to memory footprint issue?
In any case can you add little bit more information so we can know the exact issue?

Reply:
Updated.